ci-operator/config/openshift/rosa/openshift-rosa-master__e2e.yaml

```diff
@@ -26,6 +26,7 @@ tests:
     env:
       CHANNEL_GROUP: stable
       IMPORTANCE: (Critical,High)
+      NAME_PREFIX: rosa-sts-f3
```

Comment:
As same as the presubmit jobs, do we need different names for the periodic jobs?

Reply:
I prefer to have f3/f7 in the name prefix. So I can know if the leak is caused by critical+high jobs or medium+low jobs and the leak frequency.